src/command.js

```diff
@@ -70,7 +70,7 @@ export default function () {
             userType = typeof userType === 'string'
                      ? userType
                      : (tags.subscriber ? 'subscriber' : 'viewer')
-            if (!(userLevels[userType] >= userLevels[opts.rank])) {
+            if (userLevels[userType] < userLevels[opts.rank]) {
```

Comment:
I'll revert that, Atom just ran JSHint over this and it complained so I fixed it, but it doesn't really matter anyways.